Issue 63 user provided service

jmcarp opened the pull request:
Get credentials from user-provided service.

* Document user-provided service
* Set basic auth credentials from service
* Initialize new relic manually from service
* Clean up a few flake8 warnings

Notes:
* It's kind of ugly, but setting the basic auth credentials in `settings.py` seems to work. The issue is that regulations-core tries to pull the creds from the environment, but this patch proposes to get them from a user-provided service instead.
* I don't have permission to deploy this, so it wouldn't be entirely surprising if it doesn't quite work. If this looks roughly correct and I can get permission, I'll verify that it actually works.

@cmc333333

cmc333333 commented on manifest.yml:
```diff
@@ -1,6 +1,9 @@
 ---
-command: python manage.py refresh && python manage.py collectstatic --noinput && newrelic-admin run-program waitress-serve --port=$VCAP_APP_PORT atf_eregs.wsgi:application
+command: python manage.py refresh && python manage.py collectstatic --noinput && run-program waitress-serve --port=$VCAP_APP_PORT atf_eregs.wsgi:application
```
Probably need to remove "run-program", hey?

cmc333333 commented:
This makes sense to me; @jmcarp, you have perms to deploy now -- please stick to the `dev` space. Basic instructions here: https://github.com/18F/atf-eregs#deploying-code but we can chat in slack if it's not clear

jmcarp commented:
Updated the manifests and fixed the errant `run-program`, and the app seemed to deploy all right! We might not really need separate manifests for dev and prod, but I added them for now to try to preserve service names and URLs as close to the current configuration as possible. We can rename and potentially re-consolidate manifests.

cmc333333 commented:
Update on status per last conversation:
* README needs to be updated to emphasize that a `-f` arg is always required
* `atf-db` can't be shared as that's not the db name in production

cmc333333 commented:
Stellar stuff
